packages/termux-tools/pkg

```diff
@@ -26,21 +27,28 @@ assert_not_root() {
   fi
 }
 
+check_pkgcache(){
+  if [ -z "$(find $HOME/../usr/var/cache/apt/pkgcache.bin -mmin -60)" ]; then
```

Comment:
Replace `$HOME/../usr` with `$PREFIX`.

Reply:
done. it looks like I force-pushed it for some reason, can't remember why.